inbm/dispatcher-agent/dispatcher/schedule/schedules.py

```diff
@@ -9,17 +9,22 @@
 from typing import Optional
 
 @dataclass(init=True)
-class SingleSchedule:
-    """ Represents a SingleSchedule object from the Scheduled Manifest """
+class Schedule:
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Refactoring$</h2>**
**Suggestion:** Remove redundant init=True parameter in dataclass decorator as it is the default behavior.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
@dataclass
class Schedule:
```